control/plugin/client/grpc.go

```diff
@@ -75,6 +81,29 @@ func NewCollectorGrpcClient(address string, timeout time.Duration, pub *rsa.Publ
 	return p, nil
 }
 
+// NewStreamCollectorGrpcClient returns a stream collector gRPC client
+func NewStreamCollectorGrpcClient(
+	address string,
+	timeout time.Duration,
+	_ *rsa.PublicKey,
+	secure bool) (PluginStreamCollectorClient, error) {
```

Comment:
iirc it's legacy related to matching args with Go RPC calls. Maybe doesn't make sense.